packages/modular-scripts/src/utils/getPackageDependencies.ts

```diff
@@ -82,6 +82,7 @@ export async function getPackageDependencies(target: string): Promise<{
 
   // Package dependencies can be either local to the package or in the root package (hoisted)
   const packageDeps = Object.assign(
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
```

Comment:
Is this because `Object.create(null)` is typed as `any`?

Reply:
Yep so: ``Unsafe argument of type `any` assigned to a parameter of type `object`.``
Object.create() returns any 

what would be the cleanest way to deal with this. Using type assertion, typescript suggests:

_Don't use `Object` as a type. The `Object` type actually means "any non-nullish value", so it is marginally better than `unknown`._
- _If you want a type meaning "any object", you probably want `Record<string, unknown>` instead._
- _If you want a type meaning "any value", you probably want `unknown` instead._

so `Object.create(null) as Record<string, unknown>,` works but doesn't look very clean

Reply:
This is unfortunately a Typescript "needs proposal" that is currently open: https://github.com/microsoft/TypeScript/issues/1108

People had luck with typing is as `void` or `null`, it seems.

Reply:
Or, more correctly:

```ts
const nullObject: Record<keyof Object, never> = Object.create(null)
```